crates/taplo-cli/src/commands/format.rs

```diff
@@ -65,6 +66,92 @@ impl<E: Environment> Taplo<E> {
         Ok(())
     }
 
+    fn print_diff(path: impl AsRef<Path>, original: &str, formatted: &str) {
+        let path = path.as_ref();
+        println!("diff a/{path} b/{path}", path = path.display());
```

Comment:
The stdio is behind the `Environment` abstraction in order to support WASM. The standard output should be available here via `self.env.stdout()`, all printing should be done via that instead.

Reply:
Thanks, I wasn't aware